### ISSUE TITLE
Add Cantonal Orders

### ISSUE THREAD

**Issue by einfallstoll** [DP-3T/dp3t-app-ios-ch#328]
With the cantonal orders changing by the day, it would be useful to have another tab to see where masks are required, maximum people per room/event, and so on.

I know this is a bit out of scope, but I think it would be the right place to help people staying up-to-date and give them recent information about cantonal orders.

**Comment by mosea3** [comment]
Would be very useful. +1

**Comment by dhartmann84** [comment]
That would be very practical. +1

**Comment by r-r-liu** [comment]
Perhaps a separate app would be more appropriate.  The AlertSwiss app would be the obvious candidate, and indeed one does find therein the special measures enacted by the Bundesrat on Oct. 18 as well as those that go into effect in Aargau on Oct. 20, Jura on Oct. 23, Valais on Oct. 22.  Other cantons just provide a URL to their COVID-19 web page.  As the app seems to display only the most recent alert(s) per canton, I wouldn't be surprised to find that some these replaced in the near future by more recent  but ultimately less important alerts.  I also wouldn't be surprised if the app did support pinning alerts, but the cantonal offices that supply the information don't (know how to) use all the features.

**Comment by mosea3** [comment]
Indeed, Cantonal Orders are available on https://www.alert.swiss/. 

IMHO, providing structured infos for the users location in its language would be an additional plus feature to use swiss covid app.

**Comment by r-r-liu** [comment]
@mosea3:  I fear just the opposite might happen.  To those misinformed persons who presently reject the app because of BAG's ownership and the concomitant fear of Big Brother, such a feature might add to their befuddlement and wayward conviction.  Make it as a separate app first.  Later it could be offered also as an app plugin, if so designed from the start.

**Comment by r-r-liu** [comment]
@mosea3 

Um... just reread you last post and stumbled over "structured infos for the users location".  How are you then going to explain to the skeptics that the app is location agnostic and can't know where you are?
